src/components/application-loader/application-loader.tsx

```diff
@@ -9,8 +9,7 @@ export const ApplicationLoader: React.FC = ({ children }) => {
   const { pathname } = useLocation()
 
   const setUpTasks = useCallback(() => {
-    const baseUrl: string = window.location.pathname.replace(pathname, '') + '/'
-    console.debug('Base URL is', baseUrl)
+    const baseUrl: string = window.location.protocol + '//' + window.location.host.replace(pathname, '')
```

Comment:
Keep the old method. Otherwise we can't run the client in a subfolder

Reply:
did that